Source/Core/VideoCommon/RenderBase.h

```diff
@@ -133,6 +134,8 @@ class Renderer
 	// Max height/width
 	virtual int GetMaxTextureSize() = 0;
 
+	static Common::Event s_screenshotCompleted;
```

Comment:
`s_bScreenshot` appears to be the flag that determines whether or not a frame should be saved to a screenshot. I don't think we can get rid of it, at least not trivially.